command/host_create.go

```diff
@@ -23,6 +25,10 @@ func (c *createHost) Name() string { return "host.create" }
 
 func (c *createHost) ParseParams(params map[string]interface{}) error {
 	c.CreateHost = &apimodels.CreateHost{}
+
+	if _, ok := params["background"]; !ok {
```

Comment:
I don't follow this. Is your goal to make background default to true? That may be confusing, since Go booleans default to false. Have you written a test for this behavior?

Reply:
Yeah John and I talked about this for a while and we do want to default to true, but right let me add that to the tests.